app/views/home/index.html.erb

```diff
@@ -2,17 +2,17 @@
   <h2 class="govuk-heading-l govuk-grid-column-full">Overview</h2>
   <div class="govuk-grid-column-full">
     <div class="govuk-grid-column-one-third">
-      <div class="govuk-panel govuk-panel--confirmation dashboard-colour">
+      <div class="govuk-panel govuk-panel--confirmation dashboard-background">
         <h1 class="govuk-panel__title text-left">
           <%= @team_members.count %>
         </h1>
         <p class="govuk-panel__body text-left govuk-!-font-size-27">
-          Team Members
+          <%= link_to "Admins", team_members_path, class: "govuk-link govuk-link--no-visited-state dashboard-link-colour" %>
```

Comment:
I would keep this as Team Members to match the nav, can use Capybara's `within` method if tests were tricky with the duplication.